global_optimizer_config.json

```diff
@@ -1,75 +1,82 @@
-{
```

Comment:
All but two of the lines changed in this file are whitespace changes. Is there a good reason for that? If not please keep whitespace as it was.